src/index.ts

```diff
@@ -367,7 +375,7 @@ export type RunTargetOptions<
 export type SupportedCapability<Platform extends SupportedPlatform> = Platform extends 'android'
     ? 'wireguard' | 'root' | 'frida' | 'certificate-pinning-bypass'
     : Platform extends 'ios'
-    ? 'ssh' | 'frida'
+    ? 'ssh' | 'frida' | 'certificate-pinning-bypass'
```

Comment:
We need to document that this will permanently disable cert pinning on the whole device.

Reply:
Yes, but what would be the place for that?

Reply:
We should start documenting what the capabilities mean, anyway. Either in `SupportedCapability` or `PlatformApiOptions`.